metrics/socrata_config.py

```diff
@@ -3,5 +3,10 @@
         "file_name": "license_agreements_timeline.csv",
         "resource_id": "arae-ym9d",
         "sodapy_method": "replace",
-    }
+    },
+    "lde_site_plan_revisions": {
+        "file_name": "lde_site_plan_revisions.csv",
+        "resource_id": "bek3-sdgd",
+        "sodapy_method": "upsert",
+    },
```

Comment:
one more thought: either here or in the `queriess.py` file or perhaps the README it'd be nice to a quick comment explaining what each of these measure/why they matter